assets/tests/components/searchResults.test.tsx

```diff
@@ -30,19 +40,39 @@ const state: State = {
 
 describe("SearchResults", () => {
   test("renders no results", () => {
-    const tree = renderer
-      .create(
-        <StateDispatchProvider state={state} dispatch={jest.fn()}>
-          <SearchResults
-            vehicles={[]}
-            onClick={jest.fn()}
-            selectedVehicleId={null}
-          />
-        </StateDispatchProvider>
-      )
-      .toJSON()
+    render(
+      <StateDispatchProvider state={state} dispatch={jest.fn()}>
+        <SearchResults
+          vehicles={[]}
+          onClick={jest.fn()}
+          selectedVehicleId={null}
+        />
+      </StateDispatchProvider>
+    )
 
-    expect(tree).toMatchSnapshot()
```

Comment:
praise: Thanks for getting rid of this snapshot test!  
suggestion (non-blocking): It may be worth adding a second assertion that the text `NO SEARCH RESULTS` is rendered to match the test name